index.js

```diff
@@ -1,3 +1,4 @@
+var unescape = require('lodash/unescape');
```

Comment:
Lodash has npm modules for each function as well, if we're only using one we can install it and use it
https://www.npmjs.com/package/lodash.unescape